src/lib/docs/AccountKit/BalanceMethod.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Balance (Method)
+description: Used to retrieve the balance of an account for a specific token.
```

Comment:
If we're removing this method, should we remove this document?

Reply:
Yeah, I think we should.